toolkits/search/arcade_search/exceptions.py

```diff
@@ -0,0 +1,26 @@
+import json
+from typing import Optional
+
+from arcade.core.errors import RetryableToolError
```

Comment:
never import from arcade.core when outside of arcade core. We can achieve the same thing via `arcade.sdk.errors`